integration_tests/testsuite/test_custom.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import unittest
+from urlparse import urljoin
+
+import test_util
+
+
+class TestCustom(unittest.TestCase):
+
+    def __init__(self, url, methodName='runTest'):
+        self._base_url = url
+        self._url = url + test_util.CUSTOM_ENDPOINT
+        unittest.TestCase.__init__(self)
```

Comment:
IIRC, the reason i did it this way is because since I'm overriding `init()`, if I call `super()` I need to also override all the other methods inside the parent class. so it was cleaner to just init the parent class directly and be done with it